templates/templates/_error_login.html

```diff
@@ -0,0 +1,33 @@
+{% extends "templates/one-column_no_nav.html" %}
+{% block title %}Login error{% endblock %}
+
+
+{% block content %}
+<div class="p-strip is-deep">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align--center">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/bcdcf2c8-image-404.svg",
+          alt="",
+          height="365",
+          width="360",
+          hi_def=True,
+          loading="auto"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6 u-vertically-center">
+      <div>
+        <h1 class="p-heading--two">Something's gone wrong.</h1>
+        <p>Your cookie might have expired or your browser might be blocking cookies for this domain.</p>
+        <p>Please allow cookies for this domain and try again.</p>
```

Comment:
I remember seeing many errors like this on other sites in the past, and in every case I always had cookies enabled. So I don't think this is a very good message.

Since we're making a very specific check in the code when we raise this error, could we reword this to make it much more explicit so we have a hope of debugging it accurately?

How about:

```suggestion
        <p>Your browser session appears to be missing the expected authentication macaroon. This could be because cookies are disabled.</p>
```